ReadMe.md

```diff
@@ -87,6 +87,7 @@ The item tracker features a few options:
 
 * If Timespinner is placed in the incorrect folder it will not display the FileNotFoundMessage as it needs SDL library to do that
 * Entering a door backwards with a higher level keycard will permanently open it even when specific keycards are on
+* When Progressive Jump is enabled, collecting Hairpin/Lightwall/Sash may show a popup with the wrong item
```

Comment:
Its called Progressive Vertical Movement

Reply:
My bad; it's fixed.